.ci/updatecli/updatecli.d/update-gherkin-specs.yml

```diff
@@ -0,0 +1,82 @@
+name: update-gherkin-specs
+pipelineid: update-gherkin-specs
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: "{{ .github.branch }}"
+  apm:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.apm_repository }}"
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: "{{ .github.branch }}"
+
+sources:
+  sha:
+    kind: file
+    spec:
+      file: 'https://github.com/{{ .github.owner }}/{{ .github.apm_repository }}/commit/{{ .github.branch }}.patch'
+      matchpattern: "^From\\s([0-9a-f]{40})\\s"
+    transformers:
+      - findsubmatch:
+          pattern: "[0-9a-f]{40}"
+  pull_request:
+    kind: shell
+    dependson:
+      - sha
+    spec:
+      command: gh api /repos/{{ .github.owner }}/{{ .github.apm_repository }}/commits/{{ source "sha" }}/pulls --jq '.[].html_url'
+      environments:
+        - name: GITHUB_TOKEN
+        - name: PATH
+  agents-gherkin-specs-tarball:
+    kind: shell
+    scmid: apm
+    dependson:
+      - sha
+    spec:
+      command: tar cvzf {{ requiredEnv "GITHUB_WORKSPACE" }}/gherkin-specs.tgz .
+      environments:
+        - name: PATH
+      workdir: "{{ .specs.apm_gherkin_path }}"
+
+actions:
+  pr:
+    kind: "github/pullrequest"
+    scmid: default
+    spec:
+      automerge: false
+      draft: false
+      labels:
+        - "automation"
+      description: |-
+        ### What
+        APM agent Gherkin specs automatic sync
+
+        ### Why
+        *Changeset*
+        * {{ source "pull_request" }}
+        * https://github.com/elastic/apm/commit/{{ source "sha" }}
+      title: '[Automation] Update Gherkin specs'
+
+targets:
+  agent-gherkin-specs:
+    name: APM agent gherkin specs {{ source "sha" }}
+    scmid: default
+    disablesourceinput: true
+    kind: shell
+    spec:
+      command: 'tar -xzf {{ requiredEnv "GITHUB_WORKSPACE" }}/gherkin-specs.tgz && git --no-pager diff'
```

Comment:
what is the purpose of `git --no-pager diff`?

I have a guess. But I'm not certain. Do you think a comment about it's purpose would be beneficial?

Reply:
when using the `shell` target the output should provide some logs if there are changes to be applied:

- https://www.updatecli.io/docs/plugins/resource/shell/#_shell_target

> When the commands runs successfully (e.g. with an exit code of zero), the behavior depends on the content of the standard output:
>* If it is empty, then updatecli report a success with no changes applied.
>* Otherwise updatecli report a success with the content of the standard output as the resulting value of the change.

I can add a comment sure

Reply:
Cool. Thought so.

This gets rid of those "No changes to commit" error, right?

Reply:
what do you think now?

Reply:
No. I mean. My guess was confirmed :D